libs/ml/FFI.ml

```diff
@@ -0,0 +1,6 @@
+let wrappedConnect12 config host = 
+    TLSClient.connect () () () () () () config host
+
+let _ = Callback.register "MITLS_FFI_Config" TLSClient13.configure;
```

Comment:
It's not completely obvious what types these three callbacks have... can you add type annotations?